src/cli.ts

```diff
@@ -32,8 +32,19 @@ export async function run() {
         type: "string",
         desc: "A git tag that determines the upper bound of the range of commits",
       },
+      "next-version": {
+        type: "string",
+        desc: "The name of the next version",
+      },
     })
-    .example("lerna-changelog", "create a changelog for the changes after the latest available tag")
+    .example(
+      "lerna-changelog",
+      'create a changelog for the changes after the latest available tag, under "Unreleased" section'
+    )
+    .example(
+      'lerna-changelog --next-version="Next: 0.4.0"',
+      'create a changelog for the changes after the latest available tag, under "Next: 0.4.0" section'
+    )
```

Comment:
I'm not sure we need a dedicated example for this option. I'd prefer it if we kept it as it was before.

Reply:
ok 👍